### ISSUE TITLE
`reload_classifier` doesn't do what it says it does

### ISSUE THREAD

**Issue by leondavis1** [pennmem/ram_utils#240]
From the code: 

```
We take the final timestamped directory because in principle retrained
can be different depending on artifact detection.
```

However, the actual implementation takes the _first_ timestamped directory, rather than the final one

**Comment by mivade** [comment]
This is probably related to the logic that made its way into `cmlreaders` and was fixed in pennmem/cmlreaders#152

**Comment by leondavis1** [comment]
Closed by #244
